pkg/ccl/streamingccl/streamclient/client.go

```diff
@@ -41,26 +41,27 @@ type SubscriptionToken []byte
 // information to start a stream processor.
 type CheckpointToken []byte
 
-// Client provides a way for the stream ingestion job to consume a
-// specified stream.
-//
-// TODO(57427): The stream client does not yet support the concept of
-// generations in a stream.
+// Client is a wrapper for different kinds of clients which stream updates from
+// a tenant.
 type Client interface {
```

Comment:
Not to be too annoying, but I wonder what you think of slightly different names here.

Namely, I'm wondering if Client can stay client, but this becomes something like `type Dialer interface`. From a brief look if it is only needed by the getFirst.. functions, I'm not sure it needs Close() even.

Reply:
Good idea. changed the names. I kept `Close()` in dialer because `Client` and `SpanConfigClient` both use it.